GetandTake/Models/Base/BaseEntity.cs

```diff
@@ -0,0 +1,8 @@
+﻿using GetandTake.Models.BaseModels;
+
+namespace GetandTake.Models.Base
+{
+    public class BaseEntity:IBaseEntity
```

Comment:
Let's add empty spaces between `:` to make it easier to read, like
```suggestion
    public class BaseEntity : IBaseEntity
```

BTW, it is necessary to have interface? Guess we can make just this class as single entrypoint and mark it with `abstract` modifier. Same case for interfaces that are used only for DTO/entities